src/SmartComponents/Recs/helpers.js

```diff
@@ -72,3 +88,72 @@ export const bulkHostActions = async ({
     });
   }
 };
+
+const DownloadExecReport = ({ isDisabled }) => {
```

Comment:
Since this is a component, it could also live in its own file, maybe under `./components`.